cpu/stm32_common/include/stmclk.h

```diff
@@ -48,6 +48,13 @@ void stmclk_init_sysclk(void);
  */
 void stmclk_enable_hsi(void);
 
+/**
+ * @brief       Switch to multispeed internal clock (MSI)
+ * @param[in]   msi_range MSI frequency range
+ * @param[in]   ahb_divider AHB bus divider
+ */
+void stmclk_switch_msi(uint32_t msi_range, uint32_t ahb_divider);
```

Comment:
if MSI is not present anymore this is not needed.

Reply:
needed for #8403 to switch to low-power sleep mode

Reply:
Hmmm yeah but that's independent of this PR, so it should be included either in a separated (upcoming) PR or in #8493, but I'd say in a different PR enhancing this one.